sisyphus.js

```diff
@@ -7,522 +7,561 @@
+                    this.targets = this.targets || [];
+                    if ( self.options.name ) {
+                        this.href = self.options.name
+                    } else {
+                        this.href = location.hostname + location.pathname + location.search + location.hash;
+                    }
+                    this.targets = $.merge( this.targets, targets );
+                    this.targets = $.unique( this.targets );
+                    this.targets = $( this.targets );
+                    if ( ! this.browserStorage.isAvailable() ) {
+                        return false;
+                    }
+
+                    var callback_result = self.options.onBeforeRestore.call( self );
+                    if ( callback_result === undefined || callback_result ) {
+                        self.restoreAllData();
+                    }
+
+                    if ( this.options.autoRelease ) {
+                        self.bindReleaseData();
+                    }
+
+                    if ( ! params.started[ this.getInstanceIdentifier() ] ) {
+                        if ( self.isCKEditorPresent() ) {
+                            var intervalId = setInterval( function() {
+                                if (CKEDITOR.isLoaded) {
+                                    clearInterval(intervalId);
+                                    self.bindSaveData();
+                                    params.started[ self.getInstanceIdentifier() ] = true;
+                                }
+                            }, 100);
+                        } else if ( self.isTinyMCEPresent() ) {
+                            var intervalId = setInterval( function() {
+                                if (tinymce.EditorManager.editors.length != 0) {
+                                    tinymce.isLoaded = true;
+                                }
+                                if (tinymce.isLoaded) {
+                                    clearInterval(intervalId);
+                                    self.bindSaveData();
+                                    params.started[ self.getInstanceIdentifier() ] = true;
+                                }
+                            }, 100);
+                        } else {
+                            self.bindSaveData();
+                            params.started[ self.getInstanceIdentifier() ] = true;
+                        }
+                    }
+                },
+
+                isCKEditorPresent: function() {
+                    if ( this.isCKEditorExists() ) {
+                        CKEDITOR.isLoaded = false;
+                        CKEDITOR.on('instanceReady', function() {
+                            CKEDITOR.isLoaded = true;
+                        } );
+                        return true;
+                    } else {
+                        return false;
+                    }
+                },
+
+                isTinyMCEPresent: function() {
```

Comment:
This function seems redundant because it returns value of `this.isTinyMCEExists()`.